stable/openvpn/templates/openvpn-deployment.yaml

```diff
@@ -79,6 +79,10 @@ spec:
             path: "pki/issued/server.crt"
           - key: "dh.pem"
             path: "pki/dh.pem"
+          {{- if .Values.openvpn.taKey }}
```

Comment:
Reading through your implementation, it looks like the `taKey` will only apply if `openvpn.keystoreSecret` is also provided (and the `taKey` has to be previously stored in that secret). Is that right? If so, can you document it accordingly?

Reply:
That is not the case. If no secret is provided, the setup-certs.sh script, in config-openvpn.yaml, will also generate the ta.key and store it along with the other certificates.

Reply:
Ah I see - so the value `taKey` is a boolean, and you’ll generate the ta.key.

Do `taKey` and `cipher` depend on each other, or can they be sent independently? It looks like they’re both optional and independent (which is fine).

Reply:
They are both optional and independent. I included them in the same merge, because they are often used in conjunction and they are both about hardening. But it's perfectly fine to use one and not the other.